Src/Base/AMReX_ParallelDescriptor.cpp

```diff
@@ -305,7 +305,23 @@ ParallelDescriptor::StartParallel (int*    argc,
     MPI_Initialized(&sflag);
 
     if ( ! sflag) {
-	MPI_Init(argc, argv);
+
+#ifdef AMREX_MPI_MULTIPLE
+        int requested = MPI_THREAD_MULTIPLE;
+        int provided = -1;
+
+        MPI_Init_thread(argc, argv, requested, &provided);
+
+        if (provided < requested)
+        {
+            std::cout << "MPI provided < requested: " + std::to_string(provided) + 
```

Comment:
This works of course.  But why not simply `std::cout << "xxx " << int << "yyy "?